### PR TITLE
bugfix: zenko 1715 mongo upgrade

### PR DESCRIPTION
**Pull request by ssalaues**
**What does this PR do, and why do we need it?**
Adds additional check to prevent a false positive upgrade. Also adds more info on the upgrade procedure.

**Which issue does this PR fix?**
ZENKO-1715

**Special notes for your reviewers**:

### CODE REVIEW

File: kubernetes/zenko/templates/maintenance/mongo-capabilites.yaml [hunk 1]
```diff
@@ -58,6 +58,12 @@ spec:
                    sleep(10000);
                }
                db.adminCommand({ setFeatureCompatibilityVersion: "3.6" });
+               sleep(10000);
+               version = db.adminCommand({ getParameter: 1, featureCompatibilityVersion: 1 }).featureCompatibilityVersion.version;
+               if (version != '3.6'){
+                   print("Unable to upgrade capabilities, manual upgrade man be neccessary")
```
Comment (nicolas2bert) [inline kubernetes/zenko/templates/maintenance/mongo-capabilites.yaml]: typo ? `may be`

**Review by nicolas2bert — APPROVED** [review]

File: kubernetes/zenko/templates/maintenance/mongo-capabilites.yaml [hunk 2]
```diff
@@ -58,6 +58,12 @@ spec:
...
+                   quit(1)
```
Comment (JianqinWang) [inline kubernetes/zenko/templates/maintenance/mongo-capabilites.yaml]: do we need `;` here

File: kubernetes/zenko/templates/maintenance/mongo-capabilites.yaml (hunk 1 above)
Comment (ssalaues) [reply kubernetes/zenko/templates/maintenance/mongo-capabilites.yaml]: Oops! You're right, thank you

File: kubernetes/zenko/templates/maintenance/mongo-capabilites.yaml (hunk 2 above)
Comment (ssalaues) [reply kubernetes/zenko/templates/maintenance/mongo-capabilites.yaml]: My bad, I'll fix that.

**Review by JianqinWang — APPROVED** [review]

**Comment by ssalaues** [comment]
@bert-e approve

**Pull request merged**